src/ICONNameService.js

```diff
@@ -1,18 +1,44 @@
 "use strict"
+import {
+  registryInit,
+  getResolver
+} from "./helper/ins/registryService";
+import {
+  resolverInit,
+  getAddress,
+  getContent,
+  getMultihash
+} from "./helper/ins/resolverService";
 
 class ICONNameService {
   // Provider URL
-  constructor(restURL) {
-    this.restURL = restURL
+  constructor(restURL, networkId, walletAddr) {
```

Comment:
constructor(restURL, networkId, walletAddr = '<ICON_ADDRESS>')

Reply:
In `BNS.js`, wallet address is initialized with default value if user doesn't provide
```js
if (config && config.walletAddr && config.walletAddr !== "") {
      this.walletAddr = config.walletAddr;
} else {
      this.walletAddr = "hxbe258ceb872e08851f1f59694dac2558708ece11";
}
```